src/snowman.js

```diff
@@ -15,26 +78,145 @@ function getRandomWord() {
   return dictionary[index];
 }
 
-/*
-  This function will run your game. Everything you want to happen in your game should happen inside of here.
+ // This line of code gets a random word. The `word` variable will be a string.
+  const word = getRandomWord();
 
-  You should still define other, smaller functions outside of the `run()` function that have a single specific purpose, such as getting user input or checking if a guess is correct. You can then call these helper functions from inside the `run()` function.
+  //global variables
+let guessedWord = [] // stores the guessed letters forming the secret word
+let guessedLetter = '' //letter guessed to form the word
+//let WordLength = word.length +  3; 
+let WordLength = process.argv[2]; // maximum number of chances to play the game
```

Comment:
This isn't working the way you want it to. Process.argv gives us a value at the 2nd index when the user enters a something in the command line, however, because they do not enter anything aside from "node snowman.js" when they first run the file, process.argv[2] is undefine causing the user to immediately lose the game because you while loop in the run function is never accessed.

Reply:
Based on the notes you have of how you wanted wordlength to behave, you can use word.length+3 and your game works perfectly.

Reply:
Handled